src/Components/Pages/TransferPage.tsx

```diff
@@ -411,6 +416,7 @@ const TransferPage = () => {
     checkSupplies,
   } = useChainbridge();
 
+  const [isPasteDestinationAddressManually, setIsPasteDestinationAddressManually] = useState(false);
```

Comment:
The name is confusing. What do you think about `shouldPasteDestionationAddressManually`?

Reply:
Yes. It's deferentially better.